init.lua

```diff
@@ -67,6 +67,8 @@ PaperWM.default_hotkeys = {
     cycle_height    = { { "ctrl", "alt", "cmd", "shift" }, "r" },
     slurp_in        = { { "ctrl", "alt", "cmd" }, "i" },
     barf_out        = { { "ctrl", "alt", "cmd" }, "o" },
+    switch_space_l  = { { "ctrl", "alt", "cmd" }, "j" },
+    switch_space_r  = { { "ctrl", "alt", "cmd" }, "k" },
```

Comment:
We should use <kbd>cmd</kbd>+<kbd>`</kbd> (aka ⌘ + above tab), like Gnome PaperWM does. I'd avoid the PageUp / PageDn shortcuts because most Mac keyboards don't have those keys.

Reply:
Hmm, I forgot that PaperWM.spoon uses a default modifier of <kbd>ctrl</kbd><kbd>alt</kbd><kbd>cmd</kbd> a shortcut of
<kbd>ctrl</kbd><kbd>alt</kbd><kbd>cmd</kbd>+<kbd>`</kbd>

or

<kbd>ctrl</kbd><kbd>alt</kbd><kbd>cmd</kbd>+<kbd>shift</kbd><kbd>`</kbd>

Is really a handful. I expect most users remap the default shortcuts, but I want to provide default shortcuts that don't overlap with any system defaults.

Also <kbd>cmd</kbd>+<kbd>`</kbd> is supposed to switch between recently used spaces, and this feature is cycling through spaces by location. So maybe the PaperWM keyboard shortcut doesn't apply.

I personally like using <kbd>cmd</kbd>+<kbd><</kbd> for switch space left and <kbd>cmd</kbd>+<kbd>></kbd> for switch space right since it looks like a right and left arrow. Unfortunately this conflicts with PaperWM's switch next window and switch previous window.

I'm undecided on what to do...

Note: OG PaperWM avoids using Vim shortcuts, because it's not related to Vim at all and Vim shortcuts can be arbitrary and confusing to someone who's never used Vim.